package-lock.json

```diff
@@ -189,7 +189,7 @@
     },
     "libs/sdk": {
       "name": "@rudderstack/rudder-sdk-react-native",
-      "version": "1.11.0",
+      "version": "1.11.1",
```

Comment:
hm.... I wonder with this happened here? this should have been bumped when we created the release branch for 1.11.1

Reply:
This is correct. After running `npm i` it has just bumped up the already released version, please refer [here](https://www.npmjs.com/package/@rudderstack/rudder-sdk-react-native?activeTab=versions).
1.11.1 is the latest version of the React Native SDK.

Reply:
As of now, when we create the release branch, it only bumps up the version in the pacakge.json and other places except the package-lock.json file.

Reply:
we can fix that, ping me on thins next time you want to prepare a release branch